@ryansb this check fail:
```
2017-03-30 23:17:15 ERROR: Found 1 pep8 issue(s) which need to be resolved:
2017-03-30 23:17:15 ERROR: test/sanity/pep8/legacy-files.txt:358:1: A201 Remove "lib/ansible/modules/cloud/pubnub/pubnub_blocks.py" since it passes the current rule set (75%)
```

And there is no information about what exactly is wrong. How this should be fixed?

Thanks.

--- a/test/sanity/pep8/legacy-files.txt
+++ b/test/sanity/pep8/legacy-files.txt
@@ -355,7 +355,6 @@ lib/ansible/modules/cloud/profitbricks/profitbricks_datacenter.py
 lib/ansible/modules/cloud/profitbricks/profitbricks_nic.py
 lib/ansible/modules/cloud/profitbricks/profitbricks_volume.py
 lib/ansible/modules/cloud/profitbricks/profitbricks_volume_attachments.py
-lib/ansible/modules/cloud/pubnub/pubnub_blocks.py
 lib/ansible/modules/cloud/rackspace/rax_cbs_attachments.py
 lib/ansible/modules/cloud/rackspace/rax_cdb.py
 lib/ansible/modules/cloud/rackspace/rax_clb.py

